docs/en/usage/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Usage
+---
+To use a repository as a template you must simply add a `template.yml` file to the root directory on the main branch. 
+This `template.yml` defines which files should be copied from the template to the target repository 
+and if their content should be templated using a template engine. 
+
+The default template engine is [mustache](https://mustache.github.io/) if no engine is defined.
+
+![Template](assets/template.png)
+
+After a repository got marked as template it is available on the repository creation if you check `initialize repository`.
+
+![Create repository](assets/create-repo.png)
```

Comment:
We need a description of the template.yaml format. Perhaps we could create a separate page for the template.yaml syntax and link this page from the editor (scm-editor-plugin). What do you think?